veda-apis.qmd

```diff
@@ -8,15 +8,15 @@ The VEDA API source code is currently stored in a private repository: [delta-bac
 
 ### Staging (maintenance will be announced):
 
-* STAC browser (experimental): [delta-staging-stac-browser](http://delta-staging-stac-browser.s3-website-us-east-1.amazonaws.com/)
+* STAC browser (experimental): [veda-staging-stac-browser](http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/)
```

Comment:
I think we can now drop `(experimental)` here and below because v3 was officially released